utilities/copyoffload_migration.py

```
@@ -203,6 +215,176 @@ def merge_storage_secret_extra(
+        """Capture populate pod logs for one migration status poll.
+
+        Args:
+            status (str): Current migration status from migration polling.
+        """
+        if status == Plan.Status.EXECUTING:
+            try:
+                if not cached_migration_uid:
+                    migration = get_migration_for_plan(plan=plan)
+                    migration_uid = migration.instance.metadata.uid
+                    if not migration_uid:
+                        raise ValueError("Migration CR has no UID")
```

Ruff (0.15.20)

[warning] Abstract `raise` to an inner function

(TRY301)